src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -43,7 +44,41 @@ public static Channel CreateChannel(this OtlpExporterOptions options)
             }
 
 #if NETSTANDARD2_1 || NET6_0_OR_GREATER
-            return GrpcChannel.ForAddress(options.Endpoint);
+            // The specification does not dictate the parameters of the default retry policy.
+            // These defaults are borrowed from opentelemetry-java.
+            var retryPolicy = new RetryPolicy
```

Comment:
how does this impact the default [deadline](https://github.com/open-telemetry/opentelemetry-dotnet/blob/b3ffb46e3220f3751defec3ed6bfa973292e9bfb/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpGrpcMetricsExportClient.cs#L44) we set?

Is it possible that while retrying the deadline is exceeded and the retry does not complete or not performed(if retryAfter is more than default deadline)